src/main/java/org/broadinstitute/hellbender/tools/picard/illumina/ExtractIlluminaBarcodes.java

```diff
@@ -581,55 +520,52 @@ public PerTileBarcodeExtractor(
         }
 
         // These methods return the results of the extraction
-        public synchronized Map<String, BarcodeMetric> getMetrics() {
+        public Map<String, BarcodeMetric> getMetrics() {
             return this.metrics;
         }
 
-        public synchronized BarcodeMetric getNoMatchMetric() {
+        public BarcodeMetric getNoMatchMetric() {
             return this.noMatch;
         }
 
-        public synchronized Exception getException() {
+        public Exception getException() {
```

Comment:
I think this weird exception hiding can probably be removed now.  I suspect it was implemented because `Runnable` doesn't throw `Exception`